integration-tests/tests/utils/package_utils.py

```diff
@@ -0,0 +1,116 @@
+"""Provides utility functions related to the clp-package used across `integration-tests`."""
+
+import shutil
+import subprocess
+
+from clp_py_utils.clp_config import (
+    CLPConfig,
+)
+from pydantic import ValidationError
+
+from tests.utils.clp_mode_utils import compute_mode_signature, get_clp_config_from_mode
```

Comment:
```suggestion
from tests.utils.clp_mode_utils import (
    compute_mode_signature,
    get_clp_config_from_mode,
)
```
just for consistency

Reply:
agreed